README.md

```diff
@@ -38,6 +38,32 @@ OpenColorIO is free and open source software ([LICENSE](LICENSE)), and
 one of several projects actvively sponsored by the ASWF
 ([Academy Software Foundation](https://www.aswf.io/)).
 
+OpenColorIO Project Mission
+---------------------------
+
+The OpenColorIO project is committed to providing an industry standard solution 
+for highly precise, performant, and consistent color mangement across digital 
+content creation applications and pipelines.
+
+OpenColorIO aims to:
+
+* be stable, secure, and thouroughly tested on all platforms
```

Comment:
"all platforms" --> "Linux, Mac, and WIndows"

(Just to be a little more specific since IOS etc. are not currently a priority.)

Reply:
Great point.

Reply:
Changed